addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -86,11 +86,42 @@ $o-wslides-fs-side-width: 300px;
     .o_wslides_js_lesson_quiz {
         i.o_wslides_js_quiz_icon {
             cursor: pointer;
+            @media (max-width: 600px){
+                margin-right: inherit;
```

Comment:
I usually don't like "inherit" because it's not very obvious what it applies but here it seems to do the trick and I have no better proposals.